.github/workflows/stale.yml

```diff
@@ -0,0 +1,47 @@
+name: Stale
+
+# yamllint disable-line rule:truthy
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      # The 90 day stale policy
+      # Used for:
+      # - Issues & PRs
+      # - No PRs marked as no-stale or pinned
+      # - No issues marked as no-stale, help-wanted or pinned
+      - name: 90 days stale issues & PRs policy
+        uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 90
```

Comment:
This was 60 before

Reply:
Yes, I moved to 90 days on purpose as 60 days go by fast :) Also aligns with Core.